docs/components.md

```diff
@@ -17,6 +17,7 @@ The distribution offers support for the following components.
 | [awscontainerinsights](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/awscontainerinsightreceiver)      | [beta]           |
 | [awsecscontainermetrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/awsecscontainermetricsreceiver) | [beta]           |
```

Comment:
nit: not from your change, but, should be in alphabetical order by component name.

Reply:
Same for the actual go source code.

Reply:
I saw a few of these are out of order and was planning another PR to fix. I'll do that now in another PR, and rebase once the other once gets merged. 👍

Reply:
https://github.com/signalfx/splunk-otel-collector/pull/5321